example/App.js

```diff
@@ -2,15 +2,15 @@ import React, { Component } from 'react';
 import { FlatList, StatusBar, StyleSheet, Text, TouchableOpacity, View } from 'react-native';
 import { BrightcovePlayer, BrightcovePlayerPoster, BrightcovePlayerUtil } from 'react-native-brightcove-player';
 
-const ACCOUNT_ID = '5434391461001';
-const POLICY_KEY =
-  'BCpkADawqM0T8lW3nMChuAbrcunBBHmh4YkNl5e6ZrKQwPiK_Y83RAOF4DP5tyBF_ONBVgrEjqW6fbV0nKRuHvjRU3E8jdT9WMTOXfJODoPML6NUDCYTwTHxtNlr5YdyGYaCPLhMUZ3Xu61L';
-const PLAYLIST_REF_ID = 'brightcove-native-sdk-plist';
-
-// const ACCOUNT_ID = '6111983544001';
+// const ACCOUNT_ID = '5434391461001';
 // const POLICY_KEY =
-//   'BCpkADawqM3mIw2gEyYJ26qTlDZpBBB7G56vaHlkPDKmwpUQHjsi42l3vvySqov0xFDSn4cLxMKyKAIjHEj3FIv3DUP-t2gKRiNpM0Ag2g7k7aHjbjrv0N5w2ha-0RZTYQ3yqxB1kMgqi0Vu';
-// const PLAYLIST_REF_ID = '12345';
+//   'BCpkADawqM0T8lW3nMChuAbrcunBBHmh4YkNl5e6ZrKQwPiK_Y83RAOF4DP5tyBF_ONBVgrEjqW6fbV0nKRuHvjRU3E8jdT9WMTOXfJODoPML6NUDCYTwTHxtNlr5YdyGYaCPLhMUZ3Xu61L';
+// const PLAYLIST_REF_ID = 'brightcove-native-sdk-plist';
+
+const ACCOUNT_ID = '6111983544001';
+const POLICY_KEY =
+  'BCpkADawqM3mIw2gEyYJ26qTlDZpBBB7G56vaHlkPDKmwpUQHjsi42l3vvySqov0xFDSn4cLxMKyKAIjHEj3FIv3DUP-t2gKRiNpM0Ag2g7k7aHjbjrv0N5w2ha-0RZTYQ3yqxB1kMgqi0Vu';
+const PLAYLIST_REF_ID = 'brightcove';
```

Comment:
dont commit those ids anywhere pls

Reply:
👍 didn't realize they were secrets

Reply:
commented out but still there

Reply:
Those are the values from the original sample project :) Not ours.

Reply:
ahh my bad :D just merged Steven's work, so later today i will pull your changes -> make sure everything is working as it used to and merge